logging/telemetryCommon.go

```diff
@@ -59,15 +59,24 @@ type TelemetryConfig struct {
 	URI                string
 	Name               string
 	GUID               string
-	MinLogLevel        logrus.Level
-	ReportHistoryLevel logrus.Level
-	FilePath           string // Path to file on disk, if any
-	ChainID            string `json:"-"`
-	SessionGUID        string `json:"-"`
+	MinLogLevel        logrus.Level `json:"-"` // these are the logrus.Level, but we can't use it directly since on logrus version 1.4.2 they added
+	ReportHistoryLevel logrus.Level `json:"-"` // text marshalers which breaks our backward compatibility.
+	FilePath           string       // Path to file on disk, if any
+	ChainID            string       `json:"-"`
+	SessionGUID        string       `json:"-"`
 	UserName           string
```

Comment:
Are these telemetry config changes appropriate for this PR?

Reply:
The downside of having a single go.mod for both the go-swagger and our code is that we need to synchronize versions of shared components. These changes are backward compatible, so I could do these without applying this PR, and have the PR applied afterward. ( but I don't think it would add much more meaningful granularity.. )

If you think that these changes need to be splitted off, please let me know and I'll create a separate PR for these.